galley/pkg/runtime/log/scope.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import "istio.io/istio/pkg/log"
+
+// Scope for logging within the Galley runtime.
+var Scope = log.RegisterScope("runtime", "Galley runtime", 0)
```

Comment:
wouldn't you still need to define that common name in a common place?

Reply:
Possibly. In the case of MCP, the name was short enough that sharing via convention seemed reasonable. For longer names it might make sense to use a constant. Either way, the actual call site usage would be scope.Infof instead of log.Scope.Infof.

Reply:
Convention seems error prone. I'd prefer using a shared scope or a shared constant for the name.

An alternative is to import the log package as "." so we can leave off the "log." prefix.

Reply:
Shared constant or "import ." would work. I think I have a slight preference for using the constant as it makes the log usage consistent with other parts of Istio

Reply:
The issue there is then it wouldn't be consistent with the rest of Gally, which appears to be using specific scopes throughout. WRT consistency, I think consistency within galley trumps consistency with Istio.  We can always go back across Galley with a "consistency" PR to address this.

Reply:
I think we use `scope.Infof` through Galley and MCP. So long as the code here follows that convention I think we're okay. Shared scope would work and can be abstracted from the callsites, e.g.

```golang
var scope = log.Scope
```

Any change we might make later would be contained to this one spot. The callsites wouldn't need to be updated.